Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/adapter/proxy/AdapterPatientDiscoveryProxyWebServiceHelper.java

```diff
@@ -66,7 +66,7 @@ public PRPAIN201306UV02 formatGatewayResponsePRPAIN201305UV02(PRPAIN201305UV02 b
             if (body != null) {
                 LOG.debug("Before target system URL look up.");
                 url = oProxyHelper.getAdapterEndPointFromConnectionManager(sServiceName);
-                LOG.debug("After target system URL look up. URL for service{} ", sServiceName, "and url= ", url);
+                LOG.debug("After target system URL look up. URL for service: {} ", sServiceName, "and url : {}  ", url);
```

Comment:
LOG.debug("After target system URL look up. URL for service: {} and url : {}  ", sServiceName, url);

combine the messages and separate the parameters